bin/populate_m4_db.sh

```diff
@@ -160,7 +160,7 @@ for dir in "${DIRS[@]}" ; do
       fi
     done
 
-    [[ ${do_serial_check} == 0 ]] && batch_dirs+=( "${temp}" )
+    batch_dirs+=( "${temp}" )
```

Comment:
I suppose we could make sure that we found at least one file but not sure it's worth it...

Reply:
Added a TODO for it.